web/models/notes.ex

```diff
@@ -7,4 +7,10 @@ defmodule Notox.Note do
 
     timestamps
   end
+
+  def changeset(model, params \\ %{}) do
+    model
+    |> cast(params, [:title ,:description])
```

Comment:
Maybe consider extracting this to a module attribute (constant)? Also, you've got a misplaced constant.